### ISSUE TITLE
trigger at random time frame

### ISSUE THREAD

**Issue by cilerler** [fluentscheduler/FluentScheduler#175]
Is there a way to trigger a task in random time frame?

```
Schedule<MyJob>().ToRunNow().AndEvery(3).Minutes().Random();
```

## Expected behaviour

If the time is `8:20:00`;
It should run randomly between `8:20:00`-`8:22:59` which is the **1st trigger**
Then it should run second time between `8:23:00`-`8:25:59` which is the **2nd trigger**

> _If it ran at `8:22:00` for the 1st time, next time frame should be `8:22:00`-`8:24:59`_

**Comment by tallesl** [comment]
I don't think that this feature it's a good idea, it seems rather confusing (your "should" vs "should not" example is far from obvious).

If you don't mind me asking, what's the use case for this?

**Comment by cilerler** [comment]
I really believe it is important to have.
I said should because it should be implementation decision.  I just tried to give you default behaviour.

A basic scenario would be web scraping.

**Comment by tallesl** [comment]
I'll leave here as "new feature", but I'm still not convinced.
